<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Alinka" name="testCases.CreateOneProduct" tests="3" failures="1" timestamp="11 Dec 2022 20:18:02 GMT" time="4.404" errors="0">
  <testcase name="createOneProduct" time="2.263" classname="testCases.CreateOneProduct"/>
  <testcase name="readAllProducts" time="1.505" classname="testCases.CreateOneProduct"/>
  <testcase name="readOneProduct" time="0.636" classname="testCases.CreateOneProduct">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Product names are a not mathchin! expected [Amazing Headset 1.0 By MD] but found [null],
	expected [The best Headset for amazing programmers.] but found [null],
	expected [199] but found [null]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Product names are a not mathchin! expected [Amazing Headset 1.0 By MD] but found [null],
expected [The best Headset for amazing programmers.] but found [null],
expected [199] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testCases.CreateOneProduct.readOneProduct(CreateOneProduct.java:161)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- readOneProduct -->
</testsuite> <!-- testCases.CreateOneProduct -->
